pkg/apis/extensions/v1alpha1/types_bastion.go

```diff
@@ -80,7 +80,8 @@ type BastionStatus struct {
 	// DefaultStatus is a structure containing common fields used by all extension resources.
 	DefaultStatus `json:",inline"`
 	// Ingress is the external IP and/or hostname of the bastion host.
-	Ingress corev1.LoadBalancerIngress `json:"ingress"`
+	// +optional
+	Ingress corev1.LoadBalancerIngress `json:"ingress,omitempty"`
```

Comment:
This should be a pointer, right? https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required
```suggestion
	Ingress *corev1.LoadBalancerIngress `json:"ingress,omitempty"`
```

Reply:
Yes right, thanks! I incorporated it in the first commit (and second commit for the generation part). I have also added a breaking change release note for `dependecy`